website/docs/.vitepress/theme/VPNavBarSearchButton.vue

```diff
@@ -0,0 +1,216 @@
+<script lang="ts" setup>
+import type { ButtonTranslations } from 'vitepress/types/local-search'
+import { createSearchTranslate } from 'vitepress/dist/client/theme-default/support/translation'
+
+// Button-Translations
+const defaultTranslations: { button: ButtonTranslations } = {
+  button: {
+    buttonText: 'Search',
+    buttonAriaLabel: 'Search'
+  }
+}
+
+const translate = createSearchTranslate(defaultTranslations)
+</script>
+
+<template>
+  <search>
```

Comment:
```suggestion
  <search role="search">
```
seems that we need this redundancy for full support 
https://www.matuzo.at/blog/2023/form-and-search-landmark/ (see the "search role" section)